core/common/src/internal/format/parser/ParserOperation.kt

```diff
@@ -50,7 +50,12 @@ internal class NumberSpanParserOperation<Output>(
 
     init {
         require(consumers.all { (it.length ?: Int.MAX_VALUE) > 0 })
```

Comment:
This guard cannot be trigged by user-input, right?

Reply:
Yes, users can't form their own number consumers by hand at the time.